src/Components/Nav/MobileTopNav.jsx

```diff
@@ -0,0 +1,195 @@
+import SettingsIcon from "@material-ui/icons/Settings";
+import StyleIcon from "@material-ui/icons/Style";
+import VerifiedUserIcon from "@material-ui/icons/VerifiedUser";
+
+import useLinkCreator from "../../hooks/useLinkCreator.js";
+import { signOut } from "../../redux/Actions/login.js";
+
+import { ReactComponent as HamburgerMenuIcon } from "../../images/icons/hamburger.svg";
+import VocascanLogo from "../../images/logo/color-round.svg";
+
+import "./MobileTopNav.scss";
+
+import { useScrollBlock } from "../../hooks/useScrollBlock";
+import { pages } from "../../utils/constants";
+import NavButton from "./NavButton";
+
+const MobileTopNav = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [blockScroll, allowScroll] = useScrollBlock();
+
+  const location = useLocation();
+
+  const [showNav, setShowNav] = useState(true);
+
+  const [hamburgerOpen, setHamburgerOpen] = useState(false);
+
+  const isAdmin = useSelector((state) => state.login.user.isAdmin);
+
+  const { isValid: isLegalNoticeValid, url: legalNoticeUrl } = useLinkCreator({
+    path: pages.legalNotice,
+    electronFix: true,
+  });
+  const { isValid: isTermsAndConditionsValid, url: termsAndConditionsUrl } =
+    useLinkCreator({ path: pages.termsAndConditions });
+  const { isValid: isPrivacyPolicyValid, url: privacyPolicyUrl } =
+    useLinkCreator({ path: pages.privacyPolicy });
+
+  const triggerHamburgerMenu = useCallback(() => {
+    setHamburgerOpen(!hamburgerOpen);
+  }, [hamburgerOpen]);
+
+  const closeHamburgerMenu = useCallback(() => {
+    setHamburgerOpen(false);
+  }, []);
+
+  const listenScrollEvent = useCallback(() => {
+    if (window.scrollY > 10) {
+      setShowNav(false);
+    } else {
+      setShowNav(true);
+    }
+  }, []);
+
+  const handleLogout = useCallback(() => {
+    dispatch(signOut());
+  }, [dispatch]);
+
+  useEffect(() => {
+    window.addEventListener("scroll", listenScrollEvent);
+
+    return () => window.removeEventListener("scroll", listenScrollEvent);
+  }, [listenScrollEvent]);
+
+  useEffect(() => {
+    hamburgerOpen ? blockScroll() : allowScroll();
+  }, [allowScroll, blockScroll, hamburgerOpen]);
+
+  useEffect(() => {
+    setHamburgerOpen(false);
+  }, [location]);
+
+  return (
+    <>
+      <div
+        className={`mtn ${hamburgerOpen && "transparent"} ${
+          !showNav && "hidden"
+        }`}
+      >
+        <div className="mtn-inner">
+          <Link className="mtn-link" to="/" onClick={closeHamburgerMenu}>
+            <div className="mtn-link-inner">
+              <img src={VocascanLogo} alt="Logo" />
+              <p>Vocascan</p>
+            </div>
+          </Link>
+          <HamburgerMenuIcon
+            className="hamburger-menu"
+            onClick={triggerHamburgerMenu}
+          />
+        </div>
+      </div>
+      <div className={`mtn-menu ${hamburgerOpen ? "open" : "closed"}`}>
+        <ul className="mtn-menu-list">
+          <NavButton
```

Comment:
Can we extract these Menu items in a reusable component so that we only need to change new menu items once?

Reply:
This doesn't work that good, as you think, because the mobileTopNav has all the Button in it, whereas on the Desktop the buttons are splittet up in "TopNav" and "SideNav"

Reply:
What about exporting a list of items (text and link) with some properties to say where they should appear in which version?

Reply:
You always have to weigh up the effort involved.  That seems more complex

Reply:
But the more I think about it, the more I like the idea. But this would be a task for another PR after the next release as it is just refactoring

Reply:
Agreed, we can postpone this.